lib/decoder/PayloadDecoder.cpp

```diff
@@ -4,6 +4,8 @@
 //
 #include "PayloadDecoder.hpp"
 
+#include "mat/config.h"
```

Comment:
It probalby doesn't matter much. But it's maybe nicer to add it before the `PayloadDecoder.hpp` header. Generally the config goes first, and also in terms of alphabetic order - it should've been first, 'm' before 'P'.

Reply:
That way if `PayloadDecoder.hpp` class itself makes some assumption about the features being present and what not, it could have had extra `ifdefs` in it - and it would've respected the set of features from `config.h`

Reply:
Oh okay, I thought the header for the unit went first, but I'll change that.

Reply:
Fixed.